chef/cookbooks/hyperv/recipes/config.rb

```diff
@@ -56,12 +56,12 @@
 end
 
 %w{ OpenStackService.exe mkisofs.exe mkisofs_license.txt qemu-img.exe intl.dll libglib-2.0-0.dll libssp-0.dll zlib1.dll }.each do |bin_file|
-  cookbook_file "#{node[:openstack][:bin]}\\#{bin_file}" do
+  cookbook_file "#{node[:openstack][:bin]}/bin_file}" do
     source bin_file
   end
 end
 
-template "#{node[:openstack][:config]}\\nova.conf" do
+template "#{node[:openstack][:config].gsub!(/\\/, "/")}/nova.conf" do
```

Comment:
Please just ```gsub``` and not ```gsub!```, the latter one manipulates the node attribute but it should be just printed here.